webplugin/template/chat.html

```diff
@@ -48,8 +49,9 @@
                 if (skipRouting){
                     defaultSettings["skipRouting"] = !!skipRouting;
                 }
-                
+
                 var kommunicateSettings = {
+                    "isInputFocused" : isInputFocused == "false",
```

Comment:
why to invert the value from false to true and true to false?

Reply:
Made the changes and tested it